### DISCUSSION TITLE
You scored 5 out of a possible 6 points, please complete the assignment and try again.

### DISCUSSION THREAD

**Discussion by SomSingh23** [community/community#72896] [GitHub Education]
## I've finished my assignment, but I'm not sure why I'm receiving a message stating that I've completed 5 tasks out of 6.
![image](https://github.com/community/community/assets/91485305/5d607e6c-6e36-4b7a-a997-7443c6ff1230)

**Comment by FifthWye** [comment]
Same. Looks like someone who is responsible for Action Item development should try using Copilot to fix that issue 🙃🙃🙃.

![image](https://github.com/community/community/assets/41546028/10282a69-b26c-4e25-a3ef-cfacf0141a88)
![Screenshot 2023-10-30 010530](https://github.com/community/community/assets/41546028/2fadf7af-4ea4-4c9c-9fb6-229902c97825)

**Reply by FelipeLopez360** [reply]
add: 
print(calculateNumbers(2, 3));

in skills.js and make a commit to main.

refresh and enjoy!! ✌🏼

**Reply by dmitrykosintsev** [reply]
> add: print(calculateNumbers(2, 3));
> 
> in skills.js and make a commit to main.
> 
> refresh and enjoy!! ✌🏼

Worked well for me. Thank you!

**Comment by SomSingh23** [comment]
🤣🤣😅

**Comment by R1shabh-Gupta** [comment]
Did you find any solution?

**Comment by SomSingh23** [comment]
No, not yet.

**Comment by 8Blits** [comment]
Hello,

I was having the same issue! My solution was to edit the `skills.js` file by adding an additional print line then committed that to the repository. After waiting a few minutes (about 5 or so) I was able to click the [complete task button](https://education.github.com/experiences/primer_copilot). Hope this solution works for you!

I'm guessing that this worked because their point system is based on how many commits you've made to the repository. _Would need to do further testing to back up this claim._

**Comment by bleewn** [comment]
Response #1
